### DISCUSSION TITLE
SDRplay

### DISCUSSION THREAD

**Discussion by ve3elb** [JVital2013/goestools-win#8] [Ideas]
Hello will GOEStools for windows have support for SDRplay devices anytime soon. Currently it only supports Airspy and rtlsdr devices.

**Comment by JVital2013** [comment]
It might happen one day, but not anytime soon, and there are no prospects for it happening.

I am not interested in making this software "different" from the Linux version in any way - it needs to have feature parity to support full interoperability with the Linux version like it has now. The main readme states "In an attempt to keep this goestools fork compatible with the official Goestools for Linux repo, I will only accept pull requests and bug reports for Windows-specific issues. If you want to add a feature, open a PR on the official branch, then let me know about it. I may merge PRs from the official repo as requested." This is further elaborated on here: https://github.com/JVital2013/goestools-win/issues/3#issuecomment-1656231518

In all reality, this likely will not happen. There was an issue on the original project for this at https://github.com/pietern/goestools/issues/66 which was closed without implementation. Code for supporting SoapySDR (and therefore SDRPlay) can be found at https://github.com/pietern/goestools/pull/44, but it did not work correctly and the pull was closed - so it will not be added here.

TL;DR: If someone comes along and implements properly it on the Linux version and it gets vetted to work properly, then we look at adding it to the Windows port.

**Reply by ve3elb** [reply]
Jamie Thank you for the reply. 

I asked the question as I setup GOES system over the weekend but having issues when using RTLSDR device. 

Running SatDump with the SDRplay RSP everything works great but SatDump crashes everytime. 

Using the RTLSDR device sat is synced but no Deframer sync. Tried every setting I can but no luck with the rtlsdr. 

Then I tried using goestools for windows with no luck because the vit is to high. I just seen a post here with editing the config file I did not do that so I will try that later and see.  

If SatDump was not crashing I would have just left it alone.  
I tried SatDump on Win PCs and on Raspberry Pi4 but it's crash and closes everytime after a few mins. 

I will play around with the config file and see if I get it working.  

Again thank you for your help and great work.
